.github/workflows/must_update_changelog.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow would make sure that there are some changes on CHANGELOG.md or
+# the label "no-need-update-changelog" is tagged on the PR.
+
+name: "Must Update CHANGELOG"
+on:
+  pull_request:
+    branches:
+      - master
+      - release-*
+jobs:
+  must-update-changelog:
+    name: "Must Update CHANGELOG"
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "Skip if label exists"
+        id: "skip-if-label-exists"
+        env:
+          LABEL_EXISTS: ${{ contains(github.event.pull_request.labels.*.name, 'no-need-update-changelog') }}
+        run: |
+          if [ "${LABEL_EXISTS}" = "true" ] ; then
+            echo "no-need-update-changelog exists, skipping this check"
+            exit 0
+          fi
+      - name: "Collect changes"
+        id: "collect-changes"
+        if: ${{ ! contains(github.event.pull_request.labels.*.name, 'no-need-update-changelog') }}
```

Comment:
Why not use a global env.

Reply:
sorry, I do not get your point, please give me more hints and infomartion. 🙈

Reply:
Because the `contains(github.event.pull_request.labels.*.name, 'no-need-update-changelog')` is used for many times. We can cache the result by a global env:

```yaml
jobs:
  must-update-changelog:
    name: "Must Update CHANGELOG"
    runs-on: "ubuntu-latest"
    env:
      LABEL_EXISTS: ${{ contains(github.event.pull_request.labels.*.name, 'no-need-update-changelog') }}
```

Reply:
Updated! PTAL :D